lib/java/com/google/android/material/card/MaterialCardViewHelper.java

```diff
@@ -399,15 +406,29 @@ void setCheckedIcon(@Nullable Drawable checkedIcon) {
 
   void onMeasure(int measuredWidth, int measuredHeight) {
     if (clickableForegroundDrawable != null) {
-      int left = measuredWidth - checkedIconMargin - checkedIconSize;
+      int left = checkedIconMargin;
```

Comment:
Can be simplified as:

int left = isCheckedIconEnd() ? measuredWidth - checkedIconMargin - checkedIconSize : checkedIconMargin;
int bottom = isCheckedIconBottom() ? checkedIconMargin : measuredHeight - checkedIconMargin - checkedIconSize;

Reply:
Done